src/app/pages/Privacy.svelte

```diff
@@ -0,0 +1,170 @@
+<script>
+  import Button from "../components/Button.svelte";
+</script>
+
+<style>
+  .privacy {
+    width: 50%;
+    margin: auto;
+    margin-top: 20px;
+  }
+
+  .privacy h1 h2 h3 h4 {
+    font-family: Cabin, sans-serif;
+  }
```

Comment:
This selector is incorrect and unnecesary.

I think the intended selector is `.privacy h1, .privacy h2, .privacy h3, .privacy h4, .privacy h5, .privacy h6`; however, there a global styles that alread accomplish this, namely:

```css
/* From global.css: */
h1, h2, h3, h4, h5, h6 {
    font-family: var(--main-font), sans-serif;
}

:root {
    --main-font: Cabin;
}
```

So I'd suggest deleting this rule block altogether.

```suggestion
```